ports/esp32/modnetwork.c

```diff
@@ -398,7 +398,26 @@ STATIC mp_obj_t esp_status(size_t n_args, const mp_obj_t *args) {
             }
             return list;
         }
+        case (uintptr_t)MP_OBJ_NEW_QSTR(MP_QSTR_rssi): {
+            ESP_LOGD("wifi-status", "RSSI call");
```

Comment:
There doesn't need to be logging for simple things like this, it'll just make the log output harder to read.

Reply:
Maybe you have different boss than me :) Because that mine tolds "logging" are basics. Yes maybe I can remove this one, but whenever someone will solve "why it does not work" atleast they will know that "case" worked fine. It's on DEBUG level.

Also we log entry/exit of function as we did lot of submodules in DLL and there you are really unsure if everything worked well